manifests/00-crd-extension-console-cli.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consoleclidownloads.console.openshift.io
+  annotations:
+    displayName: ConsoleCLIDownload
+    description: Extension for configuring openshift web console command line interface (CLI) downloads.
+spec:
+  group: console.openshift.io
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  scope: Cluster
+  names:
+    plural: consoleclidownloads
+    singular: consoleclidownload
+    kind: ConsoleCLIDownload
+    listKind: ConsoleCLIDownloadList
+  additionalPrinterColumns:
+  - name: Display name
+    type: string
+    JSONPath: .spec.displayName
+  - name: Age
+    type: string
+    JSONPath: .metadata.creationTimestamp
+  - name: Description
+    type: string
+    JSONPath: .spec.description
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          description: Represents console CLI download spec
+          required:
+          - displayName
+          - description
+          - link
+          properties:
+            displayName:
+              type: string
+              description: Display name of the CLI download
+            description:
+              type: string
+              description: Description of the CLI download (can include markdown)
+            link:
+              type: object
+              description: Object that holds CLI download link details
+              required:
+              - href
```

Comment:
Should I presume `text` is defaulted here as well?
Looks like it is `Download {displayName} || link.text`

Reply:
Not sure if we need to doc that in the API resource.

Reply:
Per [original PR description](https://github.com/openshift/console/pull/1441):

![image](https://user-images.githubusercontent.com/280512/58206774-60671480-7caf-11e9-86d8-6867d558583b.png)

The `Download {displayName}` is preferred, not sure if we want this to be customizable?

Reply:
I'm assuming if an operator wants to create this resource to add an item to the list, that operator is also creating a deployment with a small file server w/a route & would want to use that route to link here?

Reply:
It's not necessarily hosted in cluster, although it could be. I think we leave that up to the operator.